### PR TITLE
Vis metadata

### PR DESCRIPTION
**Pull request by jasontk19**
## Description

* Updates the build to pull in vis metadata configs from GIBS and additional metadata from CMR based on concept ids found in the GIBS metadata
  * Only using the `conceptIds` fields from GIBS configs at this time
* Update smart handoffs to sort by collection, rather than layer.  Each collection shows the corresponding layers.
* Remove `daynight` flag from smart handoffs URLs to Earthdata Search

## Before Merge

- [ ] Change vis metadata URL to prod

@nasa-gibs/worldview

### CODE REVIEW

**Review by edplato — APPROVED** [review]
Looks/functions great! I added some minor/single line change comments.

File: config/default/common/features.json [hunk 1]
```diff
@@ -8,6 +8,9 @@
         "tour": true,
         "feedback": true,
         "previewSnapshots": true,
+        "vismetadata": {
+            "url": "https://uat.gibs.earthdata.nasa.gov/layer-metadata/v1.0/"
```
[the remark this answers is not in the document]
Comment (jasontk19) [reply config/default/common/features.json:12]: Yup, will be waiting on that before we release

File: web/js/containers/sidebar/smart-handoff.js [hunk 2]
```diff
@@ -468,21 +522,23 @@ class SmartHandoff extends Component {
...
     return (
       <div className="smart-handoff-side-panel">
 
-        <h1>Select an available layer to download:</h1>
         <div className="esd-notification">
           Downloading data will be performed using
           <a href="https://search.earthdata.nasa.gov" target="_blank" rel="noopener noreferrer"> NASA&apos;s Earthdata Search </a>
           application.
         </div>
+        <a className="help-link" onClick={showNotAvailableModal}>
```
[the remark this answers is not in the document]
Comment (jasontk19) [reply web/js/containers/sidebar/smart-handoff.js:539]: Yeah they definitely do!

**Review by minniewong — APPROVED** [review]
Looks good!

**Pull request merged**